templates/CRM/Admin/Form/PaymentProcessor.tpl

```diff
@@ -55,41 +55,41 @@
 <legend>{ts}Processor Details for Live Payments{/ts}</legend>
     <table class="form-layout-compressed">
         <tr class="crm-paymentProcessor-form-block-user_name">
-            <td class="label">{$form.user_name.label}</td><td>{$form.user_name.html} {help id=$ppTypeName|cat:'-live-user-name' title=$form.user_name.label}</td>
+            <td class="label">{$form.user_name.label}</td><td>{$form.user_name.html} {help id="$ppTypeName-live-user-name" title=$form.user_name.label}</td>
```

Comment:
I can't find a payment processor that these actually work for, but that was after I simplified, so whatever.